Expensify/App
# Update request a call page functionality to match Old Dot

### Details

cc @puneetlath 
### Fixed Issues

$ https://github.com/Expensify/App/issues/7067

### Tests

Test requesting a call without an extension
1. Open the concierge chat
2. Click the call button
3. Make sure text appears showing the current wait time
4. Click the call me button
5. Ensure the request a call page closes and you see an alert saying "Call requested"

Test requesting with a phone extension
Same as above but enter a phone extension number before submitting

Test an invalid phone extension
1. Enter any non-numeric symbols in the phone extension
2. Make sure an error message appears when clicking away.
3. Make sure you can't submit with an error message.



### QA Steps

Same as the test steps

### Tested On


### Screenshots


#### Web

![2022-01-10_16-30-13](https://user-images.githubusercontent.com/26260477/148860534-b12e9c3c-eecf-4b08-a471-02835a672e0d.png)
#### Mobile Web

![simulator_screenshot_5340C971-EF19-4E28-90CC-DC6631F35F86](https://user-images.githubusercontent.com/26260477/148860777-9b83048d-68a5-4bc5-896a-6890254fe418.png)

#### Desktop

![2022-01-10_16-35-02](https://user-images.githubusercontent.com/26260477/148860862-0d6fa2d1-0599-4e80-91e6-44a0f44702a1.png)

#### iOS

![simulator_screenshot_7E365433-5EF5-4625-801F-9F36BAE99403](https://user-images.githubusercontent.com/26260477/148860884-e5e02852-992c-4048-b830-8ed31c027453.png)

#### Android

![Screenshot_1641861696](https://user-images.githubusercontent.com/26260477/148861381-2ebb725b-b93a-43e4-a0cf-2c10a3c063d3.png)
